### ISSUE TITLE
Arparser for Arguments to demo.py

### ISSUE THREAD

**Issue by andrewk10** [andrewk10/AutoCompliance#42]
**Is your feature request related to a problem? Please describe.**
Currently arguments are parsed manually, this is a problem whenever changes need to be made in how arguments are handled / used.

**Describe the solution you'd like**
Implement the arparser library.

**Describe alternatives you've considered**
None, other than to ignore, but should be included in the codebase I feel.

**Additional context**
Based on this discussion; 
![image](https://user-images.githubusercontent.com/19553807/161435110-8a7a27d9-c8d3-4dca-9d76-9299549b5482.png)

Link; [](https://github.com/andrewk10/AutoCompliance/discussions/36)

**Comment by 0x30c4** [comment]
I've considered other [click](https://click.palletsprojects.com/en/5.x/arguments/) but I didn't used it because it would add a dependency to the application.

**Comment by 0x30c4** [comment]
I've properly implemented the argeparser would you please review it ?
Let me know if there any problem.

https://github.com/andrewk10/AutoCompliance/commit/a6c233e606611476429f01a73840e080ec389662

**Comment by andrewk10** [comment]
> I've considered other [click](https://click.palletsprojects.com/en/5.x/arguments/) but I didn't used it because it would add a dependency to the application.

Nah, no need for now, it's only a demo script for running through some back-end functionality.



> I've properly implemented the argeparser would you please review it ? Let me know if there any problem.
> 
> [a6c233e](https://github.com/andrewk10/AutoCompliance/commit/a6c233e606611476429f01a73840e080ec389662)

Well I merged in base branch changes to resolve some conflicts and cherry picked your own changes, it appears the demo/main script isn't functioning still as it should. See my screenshot below;

![image](https://user-images.githubusercontent.com/19553807/162812109-c0130059-4d31-48ab-a57a-90cd62fb403b.png)

**Comment by 0x30c4** [comment]
@andrewk10 got it. do you have discord ? or any other form of communication method will be really helpful i guess!

**Comment by andrewk10** [comment]
> @andrewk10 got it. do you have discord ? or any other form of communication method will be really helpful i guess!

I do, andrewk4#6215

**Comment by andrewk10** [comment]
Completed :)
